lib/lightning_web/live/sandbox_live/index.ex

```diff
@@ -188,13 +189,31 @@ defmodule LightningWeb.SandboxLive.Index do
               target_id: default_target && default_target.value
             })
 
+          has_diverged =
+            if default_target do
+              target_project =
+                Enum.find(
```

Comment:
@elias-ba It is possible to replace the `Enum.find` with `[target_project | _] = socket.assigns.workspace_projects` and the tests still pass - it may be an idea to tweak the test fixtures to close of this kind of 'finger trouble' if the change is not too expensive.